staked-relayer/src/status.rs

```diff
@@ -131,14 +130,21 @@ impl<B: BitcoinCoreApi + Clone, P: StakedRelayerPallet> RelayMonitor<B, P> {
 }
 
 pub async fn listen_for_blocks_stored<B: BitcoinCoreApi + Clone>(
-    btc_rpc: B,
-    polka_rpc: PolkaBtcProvider,
+    bitcoin_core: B,
+    btc_parachain: PolkaBtcProvider,
```

Comment:
Did you change the naming of of these recently? They used to be called `btc_rpc` and `provider`, by convention. I'm ok with renaming them, but if we do, we should also use the same naming throughout the rest of the repo

Reply:
Yes there was some renaming in the refactoring for jsonrpsee v2, I intend to slowly replace these throughout but I want to reduce code-shift.